rabbitmq/receive.js

```diff
@@ -71,11 +41,50 @@ wss.on("connection", ws => {
       if (deliverables.length > 0) {
         ws.send(JSON.stringify(deliverables.pop()));
       } else {
-        // webSockets[wsIndex].shouldDeliverDirectly = true;
         ws.send("empty");
       }
     } else if (message === "ready") {
       webSockets[wsIndex].shouldDeliverDirectly = true;
     }
   });
 });
+
+(async () => {
+  try {
+    const conn = await open;
+    const channel = await conn.createChannel();
+    const ok = await channel.assertQueue(queue, {
+      durable: false
+    });
+
+    if (ok) {
+      console.log(
+        " [*] RabbitMq initialized. Waiting for messages in %s. To exit press CTRL+C",
+        queue
+      );
+      channel.consume(
+        queue,
+        msg => {
+          const deliverable = handleMsg(msg);
+
+          webSockets.forEach(({ws, shouldDeliverDirectly}, index) => {
+            if (shouldDeliverDirectly) {
+              ws.send(JSON.stringify(deliverable));
+              webSockets[index].shouldDeliverDirectly = false;
+            } else {
```

Comment:
i would suggest to use the following style for curly brances:

```
if(condition){
}
else {
}
```

Same fo try/catch an anything else of a similar style